### ISSUE TITLE
How to set step indicator based on step life cycle

### ISSUE THREAD

**Issue by laisbastosbg** [badrinathvm/StepperView#91]
Hi! Is there a way to set the step indicators to change based on the life cycle of the step? I see that if I set StepLineOptions as custom it will change the line color, is there a similar thing for the step indicators?

**Comment by badrinathvm** [comment]
Yes , we can change the color for the step indicator as well.You can add a custom view as part of .addSteps([]) , can send you a code snippet shortly.
